core/src/main/scala/com/databricks/labs/remorph/parsers/FunctionBuilder.scala

```diff
@@ -236,4 +238,21 @@ object FunctionBuilder {
         ir.UnresolvedFunction(name, args, is_distinct = false, is_user_defined_function = false)
     }
   }
+
+  /**
+   * Functions can be called even if they are quoted or bracketed. This function removes the quotes and brackets.
+   * @param str
+   *   the possibly quoted function name
+   * @return
+   *   function name for use in lookup/matching
+   */
+  private def removeQuotesAndBrackets(str: String): String = {
```

Comment:
It seems doubtful that any user would name a function `'func'` which is what `['func']` would mean, and I don't know if TSQL would accept it or not (but it might). No built-in functions are constructed like that. 

But for the sake of correctness. I will remove one and only one of them.